queries.sql

```diff
@@ -21,4 +21,13 @@ SELECT animals.name, full_name FROM animals RIGHT JOIN owners ON owners.id = own
 SELECT COUNT(*), species.name FROM animals JOIN species ON species_id = species.id GROUP BY species.name;
 SELECT animals.name FROM animals JOIN species ON species_id = species.id JOIN owners ON owner_id = owners.id WHERE species_id = 2;
 SELECT animals.name FROM animals JOIN owners ON owner_id = owners.id WHERE owners.id = 5 AND escape_attempts = 0;
-SELECT COUNT(*) as total, owners.id, owners.full_name FROM animals RIGHT JOIN owners ON owners.id = owner_id GROUP BY owners.id ORDER BY total DESC LIMIT 1;
\ No newline at end of file
+SELECT COUNT(*) as total, owners.id, owners.full_name FROM animals RIGHT JOIN owners ON owners.id = owner_id GROUP BY owners.id ORDER BY total DESC LIMIT 1;
+SELECT animals.name FROM animals JOIN visits ON animals_id = animals.id WHERE vets_id = 1 ORDER BY visits.date LIMIT 1;
+SELECT COUNT(animals.name) FROM animals JOIN visits ON animals_id = animals.id WHERE vets_id = 3;
```

Comment:
- [ ] Kindly be noticed that you should use distinct in this query as it is asking for the different animals that Stephanie Mendez sees.